controllers/confidential_handler.go

```diff
@@ -57,22 +77,98 @@ func (r *KataConfigOpenShiftReconciler) handleFeatureConfidential(state FeatureG
 	}
 
 	// peer pods config
+	if r.kataConfig.Spec.EnablePeerPods {
```

Comment:
All the lines above are also specific to peer pods AFAICT. It seems this `if` should move to line 40 for now.

Reply:
No longer getting the errors from the image generator. Fixed.